src/sentry/api/serializers/rest_framework/sentry_app.py

```diff
@@ -20,4 +20,6 @@ def validate(self, data):
 class SentryAppSerializer(Serializer):
     name = serializers.CharField()
     scopes = ApiScopesField()
-    webhook_url = serializers.CharField()
+    webhook_url = serializers.URLField()
+    redirect_url = serializers.URLField(required=False)
+    overview = serializers.CharField(required=False)
```

Comment:
We'll probably want to validate that this is valid markdown eventually. Not necessary yet, just a thought.